docs/tutorials/quickstart-app-plugin.md

```diff
@@ -120,66 +120,7 @@ github-playground page. Our data accessed is synchronous. So we just grab and
 go.
 
 6. Here is the entire file for reference
-<details><summary>Complete ExampleComponent.tsx</summary>
-<p>
-
-```tsx
-import React, { FC } from 'react';
-import { Typography, Grid } from '@material-ui/core';
-import {
-  InfoCard,
-  Header,
-  Page,
-  pageTheme,
-  Content,
-  ContentHeader,
-  HeaderLabel,
-  SupportButton,
-  identityApiRef,
-} from '@backstage/core';
-import { useApi } from '@backstage/core-api';
-import ExampleFetchComponent from '../ExampleFetchComponent';
-
-const ExampleComponent: FC<{}> = () => {
-  const identityApi = useApi(identityApiRef);
-  const userId = identityApi.getUserId();
-  const profile = identityApi.getProfile();
-
-  return (
-    <Page theme={pageTheme.tool}>
-      <Header
-        title="Welcome to github-playground!"
-        subtitle="Optional subtitle"
-      >
-        <HeaderLabel label="Owner" value="Team X" />
-        <HeaderLabel label="Lifecycle" value="Alpha" />
-      </Header>
-      <Content>
-        <ContentHeader title="Plugin title">
-          <SupportButton>A description of your plugin goes here.</SupportButton>
-        </ContentHeader>
-        <Grid container spacing={3} direction="column">
-          <Grid item>
-            <InfoCard title={userId}>
-              <Typography variant="body1">
-                {`${profile.displayName} | ${profile.email}`}
-              </Typography>
-            </InfoCard>
-          </Grid>
-          <Grid item>
-            <ExampleFetchComponent />
-          </Grid>
-        </Grid>
-      </Content>
-    </Page>
-  );
-};
-
-export default ExampleComponent;
-```
-
-</p>
-</details>
+   [ExampleComponent.tsx](https://github.com/johnson-jesse/simple-backstage-app-plugin/blob/master/plugins/github-playground/src/components/ExampleComponent/ExampleComponent.md)
```

Comment:
This points to your version of Backstage.

Reply:
@stefanalund  This is actually right since it points to the example repo. But I understand if you don't want external links. Do you have a work around? A place for the example `simple-backstage-app-plugin` application to live rather than my repo? I could remove the reference altogether?

Reply:
Just FYI, we have a `/contrib` which may or may not be useful here. https://github.com/spotify/backstage/issues/2459

Reply:
Moved both `.md` files to contrib > docs > tutorials > quickstart-app-plugin and updated links.